### ISSUE TITLE
Upload crashed

### ISSUE THREAD

**Issue by andreaNCCEH** [abtassociates/eva#450]
Description:
Uploading a balance of state (NC-503)'s hashed HMIS CSV results in a crash. Message says not the date and time and to create a github note. Our hashed hmis csv worked on Monday 11.27.23.

Reproduce: 
It would take a copy of our export I believe. I simply went to upload and get an error message. I mostly recently attempted at 12:35 pm EST. I am using a desktop, mozilla firefox browser [Firefox 120 on Windows 11]. I also tried on chrome [Chrome 119 on Windows 11] with the same results.
![image](https://github.com/abtassociates/eva/assets/67340114/054ccf38-84ea-4f19-9107-fc84ccfa1a1b)

**Comment by scampillo** [comment]
CA-502 seeing the same issue.   Our hashed hmis csv worked on Wednesday 11.29.23.

**Comment by haileypate** [comment]
Got the same issue just now with an upload from Austin, Texas. We've had successful uploads in the past.

**Comment by kiadso** [comment]
Hi everyone!
Thank you all for reporting this! We did an update this morning, so even though we haven't quite figured out what's happening, we reverted Eva to the version it was this morning in case the cause of the problem is in the update. Please let us know if you're still having the issue!

**Comment by alex-silverman** [comment]
@andreaNCCEH @scampillo @haileypate  - can you all try re-uploading your files on a test app [here](https://hmis-stage.abtsites.com/eva-dev)? I think the problem may be null values in certain columns where nulls are not allowed. Normally, Eva would gracefully catch these, but due to a recent incomplete update - which is fixed in the test app - that code wasn't working.

**Comment by scampillo** [comment]
Yes, I was able to successfully upload to the test app.  I do see 4 - Nulls not allowed or incorrect data type errors.
